actions/watch.ts

```diff
@@ -73,22 +97,24 @@ async function _checkForSettlement(
       ) as [string, string, string, BigNumber, BigNumber, BigNumber, string];
 
       // Check if the owner is in the registry
-      if (registry.ownerOrders.has(owner)) {
+      if (ownerOrders.has(owner)) {
         // Get the conditionalOrders for the owner
-        const conditionalOrders = registry.ownerOrders.get(owner);
+        const conditionalOrders = ownerOrders.get(owner);
         // Iterate over the conditionalOrders and update the status of the orderUid
         conditionalOrders?.forEach((conditionalOrder) => {
           // Check if the orderUid is in the conditionalOrder
           if (conditionalOrder.orders.has(orderUid)) {
             // Update the status of the orderUid to FILLED
-            console.log(`Update order ${orderUid} to status FILLED`);
+            console.log(
+              `Update order ${orderUid} to status FILLED. Settlement Tx: ${tx}`
```

Comment:
tx hash is useful, now we can track the order to the TX that created the smart order and resulted in creating this orderUid, that now is filled :)